tests/ui-toml/toml_import_rename/clippy.toml

```diff
@@ -0,0 +1,7 @@
+import-renames = [
+    { path = "syn::path::Path", rename = "SynPath" },
+    { path = "serde::de::Deserializer", rename = "SomethingElse" },
+    { path = "serde::ser::Serializer", rename = "DarkTower" },
+    { path = "alloc::collections::btree::map::BTreeMap", rename = "Map" },
+    { path = "regex::bytes", rename = "foo" },
+]
```

Comment:
As explained in my other comment, this should work with `std::collections::BTreeMap`, `serde::Deserialize`, etc.